content/en/docs/setup/scratch.md

```diff
@@ -766,10 +767,13 @@ Template for controller manager pod:
 Flags to consider using with controller manager:
 
  - `--cluster-cidr=`, the CIDR range for pods in cluster.
+ - `--allocate-node-cidrs=true`, allow CIDRs allocated for Node to make CNI plugins work properly.
  - `--allocate-node-cidrs=`, if you are using `--cloud-provider=`, allocate and set the CIDRs for pods on the cloud provider.
  - `--cloud-provider=` and `--cloud-config` as described in apiserver section.
  - `--service-account-private-key-file=/srv/kubernetes/server.key`, used by the [service account](/docs/user-guide/service-accounts) feature.
- - `--master=127.0.0.1:8080`
+ - `--root-ca-file=/srv/kubernetes/ca.pem`, used by the [service account](/docs/user-guide/service-accounts) feature.
```

Comment:
this is fine but we should also remove the `--service-account-private-key-file` if that is now removed